builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -0,0 +1,1062 @@
+            if(line.contains('Stack Outputs')) afterStackOutputs = true // Here we are only interested in lines that follow 'Stack Outputs' marker
+          }
+
+          for(def name2ArnString: resourceArns) {
+            def name2ArnPair = name2ArnString.split(': ') // Here we rely on the space after the ': ' so it is split properly without affecting the arn itself that is full of colons but never has a space after
+            if(name2ArnPair.size() > 1) {
+              def arn = name2ArnPair[1].trim()
+              if(arn.startsWith('arn')) {
+                 def arnAsArray = arn.split(':') // Here we splitting the arn itself in hope to obtain type of the resource. Should be the third element: arn:aws:dynamodb:us-east-1:102707241671:table/jazzUsersTable53
+                 if(arnAsArray.size() > 2) { // Making sure that the third element exists
+                   def artifactType = arnAsArray[2]
+                   if(artifactType == 'kinesis') artifactType = 'kinesis_stream' // 'kinesis_stream' needs to be instaead of simple 'kinesis' that did not get persisted :-( by itself
+                   events.sendCompletedEvent('CREATE_ASSET',
+                                             null,
+                                             utilModule.generateAssetMap("aws",
+                                                                         arn,
+                                                                         artifactType,
+                                                                         config),
+                                             environment_logical_id)
+                   echo "Sending type $artifactType with arn $arn"
+
+                }
+              }
+            }
+          }
+
+
+          // reset Credentials
+          resetCredentials(credsId)
+          if (domain != "jazz") {
+            createSubscriptionFilters(stackName, configLoader.AWS.REGION, roleId);
+          }
+
+          if (domain && domain == "jazz") {
+            serviceConfigdata.setLogStreamPermission(config)
+            serviceConfigdata.setKinesisStream(config)
+          }
+          def svc_response = echoServiceInfo(environment_logical_id)
+          send_status_email(config, 'COMPLETED', svc_response)
+
+        } catch (ex) {
+          send_status_email(config, 'FAILED', '')
+          events.sendFailureEvent('UPDATE_ENVIRONMENT', ex.getMessage(), environmentDeploymentMetadata.generateEnvironmentMap("deployment_failed", environment_logical_id, null), environment_logical_id)
+          events.sendFailureEvent('UPDATE_DEPLOYMENT', ex.getMessage(), environmentDeploymentMetadata.generateDeploymentMap("failed", environment_logical_id, gitCommitHash), environment_logical_id)
+          events.sendFailureEvent('DEPLOY_TO_AWS', ex.getMessage(), context_map, environment_logical_id)
+          error ex.getMessage()
+        }
+        environmentDeploymentMetadata.setEnvironmentEndpoint(lambdaARN)
```

Comment:
Same as above. This is N/A for sls_apps.

Reply:
Resolved with https://github.com/tmobile/jazz/pull/829/commits/3db02d4a23f87f5a0ee309064407120924d0ebca